web/src/main/java/com/futurewei/alcor/web/entity/node/NodeInfo.java

```diff
@@ -32,9 +33,11 @@
     private static final Logger logger = LoggerFactory.getLogger(NodeInfo.class);
 
     @JsonProperty("node_id")
+    // @QuerySqlField(index = true)
     private String id;
 
     @JsonProperty("node_name")
+    // @QuerySqlField(index = true)
```

Comment:
@pkommoju and @DavidLiu506 Do we need index for `node_id` and `node_name`? If not, please remove it.

Reply:
node_name needs index. In fact NMM swagger and the getAllNodes allows querying on any combination of name, mac, local ip, ncm_id. I should add index on all of them. id may not be needed if getAllNodes doesn't use SCAN query if querying by only id. I will check this path and remove index on id if this is so.

Reply:
Removed index on id, added it on localIp, macAddress, ncm_id.